go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/cloudflare/cloudflared
+
+go 1.12
+
+require (
+	github.com/certifi/gocertifi v0.0.0-20190506164543-d2eda7129713 // indirect
+	github.com/cloudflare/brotli-go v0.0.0-20180507233613-18c9f6c67e3d
+	github.com/cloudflare/golibs v0.0.0-20190417125240-4efefffc6d5c
+	github.com/coredns/coredns v1.5.0
+	github.com/coreos/go-oidc v0.0.0-20171002155002-a93f71fdfe73
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
+	github.com/elgs/gosqljson v0.0.0-20160403005647-027aa4915315
+	github.com/equinox-io/equinox v1.2.0
+	github.com/facebookgo/ensure v0.0.0-20160127193407-b4ab57deab51 // indirect
+	github.com/facebookgo/freeport v0.0.0-20150612182905-d4adf43b75b9 // indirect
+	github.com/facebookgo/grace v0.0.0-20180706040059-75cf19382434
+	github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
+	github.com/facebookgo/subset v0.0.0-20150612182917-8dac2c3c4870 // indirect
+	github.com/getsentry/raven-go v0.2.0
+	github.com/golang-collections/collections v0.0.0-20130729185459-604e922904d3
+	github.com/google/uuid v1.1.1
+	github.com/gorilla/mux v1.7.1
+	github.com/gorilla/websocket v1.4.0
+	github.com/kylelemons/godebug v1.1.0 // indirect
+	github.com/lib/pq v1.1.1
+	github.com/mattn/go-colorable v0.1.1
+	github.com/miekg/dns v1.1.9
```

Comment:
Due to https://github.com/miekg/dns/issues/941 we pinned this dependency at commit version `6da3249dfb57fbaa16efafcd8744cee8809d80cd`:
https://github.com/cloudflare/cloudflared/commit/781e1fef7bafec91b242682a94b9f37719a0e788

If I understand go modules correctly, this can be accomplished by running `go get github.com/miekg/dns@6da3249dfb57fbaa16efafcd8744cee8809d80cd` and committing the result –– would you mind doing that?

Reply:
@nickvollmar https://github.com/cloudflare/cloudflared/commit/bab7583a97e8f7776640889f8b823f63a77e709a

Bumped it off of `6da3249dfb57fbaa16efafcd8744cee8809d80cd` to "=1.1.8", was that a mistake? (this is the current `master`)

In this specific case, `go get` with a pinned version would cause go modules to downgrade `coredns` dependency, This is because coredns v1.5.0 requires a newer version of `dns`.  So, to override a pin when a dependency declares a version, you need to use `replace`, like this:

```
go mod edit -replace github.com/miekg/dns=github.com/miekg/dns@6da3249dfb57fbaa16efafcd8744cee8809d80cd
```

Done in 9a3a9fc4a09d6e5abbf3f6615f51935106253bdf and 5785bb46207da525578188bfbbcd28c1a5d240dc